src/tests/update-attributes.js

```diff
@@ -0,0 +1,75 @@
+import { SharedArray } from 'k6/data';
+import { createEntity } from '../api/create-entity.js';
+import { updateAttributes } from '../api/update-attributes.js'
+import { uuidv4 } from 'https://jslib.k6.io/k6-utils/1.4.0/index.js';
+import { randomItem } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+
+export const options = {
+    vus: 10,
+    iterations: 100000,
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        update_attributes_duration: ['avg<200', 'p(95)<400']
+    },
+    setupTimeout: '5m',
```

Comment:
hmm... why?

Reply:
it was for my local test (I had done for 1000 entities, it took more time for the setup)

Reply:
so it has to be removed? we don't want the test to fail if we create many entities in the setup, no?

Reply:
Yes I remove it